include/picongpu/CMakeLists.txt

```diff
@@ -302,42 +295,43 @@ endif(PNGwriter_FOUND)
 # ISAAC
 ################################################################################
 
-find_package(ISAAC 1.3.0 CONFIG QUIET)
-if(ISAAC_FOUND)
-    message(STATUS "Found ISAAC: ${ISAAC_DIR}")
-    SET(ISAAC_STEREO "No" CACHE STRING "Using stereoscopy")
-    SET_PROPERTY(CACHE ISAAC_STEREO PROPERTY STRINGS No SideBySide Anaglyph)
-
-    if(${ISAAC_STEREO} STREQUAL "No")
-        add_definitions(-DISAAC_STEREO=0)
-    endif()
-    if(${ISAAC_STEREO} STREQUAL "SideBySide")
-        add_definitions(-DISAAC_STEREO=1)
-    endif()
-    if(${ISAAC_STEREO} STREQUAL "Anaglyph")
-        add_definitions(-DISAAC_STEREO=2)
-    endif()
-
-    include_directories(SYSTEM ${ISAAC_INCLUDE_DIRS})
-    set(LIBS ${LIBS} ${ISAAC_LIBRARIES})
-
-    set(ISAAC_MAX_FUNCTORS "3" CACHE STRING "Max length of the isaac functor chain" )
-    set(ISAAC_DEFAULT_WEIGHT "7" CACHE STRING "Default weight of an isaac source" )
-    add_definitions(${ISAAC_DEFINITIONS})
-    add_definitions(-DISAAC_MAX_FUNCTORS=${ISAAC_MAX_FUNCTORS})
-    add_definitions(-DISAAC_FUNCTOR_POW_ENABLED=0)
-    add_definitions(-DISAAC_DEFAULT_WEIGHT=${ISAAC_DEFAULT_WEIGHT})
-
-    add_definitions(-DENABLE_ISAAC=1)
-else(ISAAC_FOUND)
-    if(DEFINED ISAAC_DEPENDENCY_HINTS)
-        message(STATUS "ISAAC was found, but detected the following "
-                       "problems:" ${ISAAC_DEPENDENCY_HINTS})
-    else()
-        message(STATUS "Could NOT find ISAAC - set ISAAC_DIR or check your CMAKE_PREFIX_PATH")
-    endif()
-endif(ISAAC_FOUND)
-
+if(ALPAKA_ACC_GPU_CUDA_ENABLE)
+    find_package(ISAAC 1.3.0 CONFIG QUIET)
```

Comment:
It's not like ISAAC has full Alpaka support. -_- Did you run in problems with letting ISAAC in?